src/manager.spec.js

```diff
@@ -231,35 +231,62 @@ describe('AdManager', function() {
       TestHelper.stub(adManager.adUnits.units.header, 'onSlotRenderEnded');
       eventSpy = sinon.spy();
       adElement.addEventListener('dfpSlotRenderEnded', eventSpy);
-      adManager.onSlotRenderEnded(event);
```

Comment:
Moved this into each individual test for finer grained control. Otherwise this `beforeEach` would need to be duplicated for the case where `event` needs different properties before `onSlotRenderEnded` is executed.